integration_tests/custom_reporters/__tests__/add-fail-test.js

```diff
@@ -0,0 +1,9 @@
+const add = require('../add');
+
+describe('CustomReporters', () => {
+    test('adds fail', () => {
```

Comment:
@dmitriiabramov I did run it, for some reason `yarn prettier` only runs for the `__tests__` in the root and not for any other tests.  :confused: